x/pki/handler_test.go

```diff
@@ -335,10 +335,11 @@ func TestHandler_ProposeAddX509RootCert_ForNocCertificate(t *testing.T) {
 
```

Comment:
Do we have unit tests for the new AllCertificates logic?
To check that it correctly returns all types of certificates and is updated during remove/revoke actions?